invenio_rdm_records/services/github/release.py

```diff
@@ -55,7 +57,7 @@ def metadata(self):
                         {
                             "person_or_org": {
                                 "type": "personal",
-                                "family_name": "Unknown",
+                                "family_name": _("Unknown"),
```

Comment:
is that `Unknown` really a to translatable string? it seems like an item of a controlled vocabulary.

Reply:
Controlled vocabularies should not be translated since they serve as language-agnostic identifiers. However, in this case, the code seems to be setting a fallback value specifically for display purposes, meaning it is intended for end users. or?